server/index.js

```diff
@@ -0,0 +1,42 @@
+import http from 'http'
+import { URL } from 'url'
+
+import getBrowsers from './get-browsers.js'
+
+const DEFAULT_QUERY = 'defaults'
+const GLOBAL_REGION = 'Global'
+const PORT = process.env.PORT || 5000
+
+http
+  .createServer(async (req, res) => {
+    let url = new URL(req.url, `http://${req.headers.host}/`)
+
+    if (url.pathname === '/') {
```

Comment:
Do we plan to have a separated domains (if we use `/`)?

It is bad for performance. https://t.me/iamakulov_channel/678

I suggest putting client and server to the single Docker image to use the same domain and re-use the power of HTTP/2.

Reply:
Change endpoint to `/api`

and add TODO:
`// TODO Put client and server to the single Docker image to use the same domain and re-use the power of HTTP/2`

Reply:
What about Vercel and Github Actions for MVP?)

Reply:
I need:

1. Custom HTTP header for caching best practices
2. Ability to run custom HTTP server
3. Run static server and HTTP server on the same domain
4. No privacy issues. The company should not track users creepy.

If we can do it with Vercel, let’s use it